lib/Button/Button.less

```diff
@@ -56,9 +52,18 @@
 		border: @moon-button-border-width solid @moon-spotlight-border-color;
 		background-color: @moon-button-background-color;
 		color: @moon-button-text-color;
+
+		&.translucent:not(.disabled) {
+			background-color: fade(@moon-button-background-color, @moon-button-translucent-opacity);
+		}
+		&.transparent:not(.disabled) {
+			background-color: transparent;
+		}
 	}
 
-	&.spotlight {
+	&.spotlight,
+	&.spotlight.translucent:not(.disabled),
+	&.spotlight.transparent:not(.disabled) {
```

Comment:
65 +      &.spotlight.translucent:not(.disabled),
66 +      &.spotlight.transparent:not(.disabled) 
we need those selectors? those selectors appear again below.